Makefile

```diff
@@ -0,0 +1,113 @@
+
+LOGRUS_URL := github.com/sirupsen/logrus
+
+LOGRUS_VERSION := $(shell find go.mod -type f -exec cat {} + | grep ${LOGRUS_URL} | awk '{print $$NF}')
+
+LOGRUS_PATH := $(GOPATH)/pkg/mod/${LOGRUS_URL}\@${LOGRUS_VERSION}
+
+export GO111MODULE=on
+
+all: check-style dist
+
+## Runs govet and gofmt against all packages.
+.PHONY: check-style
+check-style: govet lint
+	@echo Checking for style guide compliance
+
+## Runs lint against all packages.
+.PHONY: lint
+lint:
+	@echo Running lint
+	env GO111MODULE=off $(GO) get -u golang.org/x/lint/golint
+	golint -set_exit_status ./...
+	@echo lint success
+
+## Runs govet against all packages.
+.PHONY: vet
+govet:
+	@echo Running govet
+	$(GO) vet ./...
+	@echo Govet success
+
+## Builds and thats all :)
+.PHONY: dist
+dist:	build
+
+.PHONY: binaries
+binaries: ## Build binaries of elrond
+	@echo Building binaries of elrond
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 $(GO) build -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/elrond-linux-amd64  ./cmd/$(APP)
+	GOOS=darwin GOARCH=amd64 CGO_ENABLED=0 $(GO) build -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/elrond-darwin-amd64  ./cmd/$(APP)
+	GOOS=linux GOARCH=arm64 CGO_ENABLED=0 $(GO) build -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/elrond-linux-arm64 ./cmd/$(APP)
+	GOOS=darwin GOARCH=arm64 CGO_ENABLED=0 $(GO) build -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/elrond-darwin-arm64  ./cmd/$(APP)
+
+.PHONY: build
+build:
+	@echo Building Elrond
+	GOOS=linux CGO_ENABLED=0 $(GO) build -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/main ./cmd/$(APP)
+
+
+.PHONY: build-image
+build-image:  ## Build the docker image for Elrond
+	@echo Building Elrond Docker Image
+	docker buildx build \
+	--platform linux/arm64,linux/amd64 \
+	--build-arg DOCKER_BUILD_IMAGE=$(DOCKER_BUILD_IMAGE) \
+	--build-arg DOCKER_BASE_IMAGE=$(DOCKER_BASE_IMAGE) \
+	. -f build/Dockerfile -t $(ELROND_IMAGE) \
+	--no-cache \
+
+.PHONY: build-image-with-tag
+build-image-with-tag:  ## Build the docker image for elrond
+	@echo Building Elrond Docker Image
+	echo $(DOCKER_PASSWORD) | docker login --username $(DOCKER_USERNAME) --password-stdin
+	docker buildx build \
+	--platform linux/arm64,linux/amd64 \
+	--build-arg DOCKER_BUILD_IMAGE=$(DOCKER_BUILD_IMAGE) \
+	--build-arg DOCKER_BASE_IMAGE=$(DOCKER_BASE_IMAGE) \
+	. -f build/Dockerfile -t $(ELROND_IMAGE) -t $(ELROND_IMAGE_REPO):${TAG} \
+	--push
+
+.PHONY: push-image-pr
+push-image-pr:
+	@echo Push Image PR
+	bash ./scripts/push-image-pr.sh
+
+.PHONY: push-image
+push-image:
+	@echo Push Image
+	bash ./scripts/push-image.sh
+
+.PHONY: install
+install: build
+	go install ./...
+
+.PHONY: release
+release:
+	@echo Cut a release
+	bash ./scripts/release.sh
```

Comment:
Since you added this you need to add the relevant script and the relevant CI logic.

Reply:
Yeap already did, just forgot to push thanks ;)